client/src/elements/schema-editor/schema-editor-table.js

```diff
@@ -1,4 +1,5 @@
 import { bindable, inject, Loader } from "aurelia-framework";
+import { I18N } from "aurelia-i18n";
```

Comment:
this is not needed here as far as I see. You can use the `t` behaviour in the template without importing i18n here. This is just needed if you want to imperatively interact with the translation system.

you can then also remove the injection.